### PR TITLE
Fix sitemap

### PR DESCRIPTION
**Pull request by bensymonds**
See commits.

### CODE REVIEW

**Comment by bensymonds** [comment]
@withassociates/with-core this is only failing tests because circleci is hooked up but not yet configured (see #150). The travis build is fine. We want to get this in before #116 so please take a look at some point!

**Comment by jgwhite** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@
 - [#101](https://github.com/withassociates/slices/pull/101): Remove `PagesHelper#google_analytics_tracking_code` and `PagesHelper#add_tracking_code`
     * You should include the GA Javascript yourself.
 
+### Bug fixes
+- [#152](https://github.com/withassociates/slices/pull/152): Fix issue where sitemap wasn't showing correct icons for page types and wasn't showing child pages in correct order
+
```
Comment (bensymonds) [inline CHANGELOG.md]: @withassociates/with-core happy with this? We can have a "bug fixes" section in parallel with "breaking changes".

**Comment by erino** [comment]
:+1:

**Pull request merged**